src/Controller/MapController.php

```diff
@@ -5,16 +5,44 @@
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\HttpFoundation\Request;
+use App\Repository\FablabRepository;
+use App\Entity\Fablab;
+use maxh\Nominatim\Nominatim;
 
+/**
+ * @Route("/map")
+*/
 class MapController extends AbstractController
 {
     /**
-     * @Route("/map", name="map")
+     * @Route("/", name="map")
      */
     public function index(): Response
     {
+        $fablabRepository = $this->getDoctrine()
+             ->getRepository(Fablab::class)
+             ->findAll();
+
+        if (!$fablabRepository) {
+            throw $this->createNotFoundException(
+                'No Fablab found in Falab\'s table.'
+            );
+        }
+
+        $url = 'http://nominatim.openstreetmap.org/';
+        $nominatim = new Nominatim($url);
+
+        $data = [];
+        for ($i = 0; $i < count($fablabRepository); $i++) {
```

Comment:
doesn't look like you need a for loop: use a foreach. or even better, give a try to array_map

Reply:
done.